### PR TITLE
DOC-6506: Added CENM 1.6.1 release notes

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/1.6/cenm/release-notes.html

### CODE REVIEW

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 1]
```diff
@@ -14,6 +14,23 @@ title: Release notes
...
 # Corda Enterprise Network Manager release notes
 
+## Corda Enterprise Network Manager 1.6.1
+
+### New features and enhancements
+
+* `sshdHost` can now be specified within the `shell {}` config block for the CENM services that support Shell.
+* Added a new `host` option in the `enmListener` configuration.
+* Added a new `host` option in the `adminListener` configuration.
+* The Zone Service can now be started with a configuration file instead of command-line flags. However, the original method of starting the Zone Service has been unaffected.
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/release-notes.md:24]: maybe (for zone/auth respectively)

The zone service can now be started either with a configuration file or with command line options, any existing deployments using the original method remain unaffected..

something like that maybe?

Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/release-notes.md]: Updated

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 2]
```diff
@@ -14,6 +14,23 @@ title: Release notes
 
 # Corda Enterprise Network Manager release notes
 
+## Corda Enterprise Network Manager 1.6.1
+
+### New features and enhancements
+
+* `sshdHost` can now be specified within the `shell {}` config block for the CENM services that support Shell.
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/release-notes.md:21]: support the interactive Shell

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 3]
```diff
@@ -14,6 +14,23 @@ title: Release notes
...
+* Added a new `host` option in the `enmListener` configuration.
+* Added a new `host` option in the `adminListener` configuration.
+* The Zone Service can now be started either with a configuration file or with command-line options. Any existing deployments using the original method remain unaffected.
+* The Angel Service can now be started either with a configuration file or with command-line options. Any existing deployments using the original method remain unaffected.
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/release-notes.md:25]: ... can now be started either with a configuration file or with the existing command-line options. Any current deployments using the original method remain unaffected.

(for both angel and zone) I think on reflection this sounds a bit better

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 4]
```diff
@@ -14,6 +14,23 @@ title: Release notes
 
 # Corda Enterprise Network Manager release notes
 
+## Corda Enterprise Network Manager 1.6.1
+
+### New features and enhancements
+
+* `sshdHost` can now be specified within the `shell {}` config block for the CENM services that support Shell.
+* Added a new `host` option in the `enmListener` configuration.
+* Added a new `host` option in the `adminListener` configuration.
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/release-notes.md:23]: Added a new optional `host` parameter in the ... configuration

File: content/en/platform/corda/1.6/cenm/release-notes.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/release-notes.md:21]: updated

File: content/en/platform/corda/1.6/cenm/release-notes.md (hunk 3 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/release-notes.md:25]: updated

File: content/en/platform/corda/1.6/cenm/release-notes.md (hunk 4 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/release-notes.md:23]: updated

File: content/en/platform/corda/1.6/cenm/release-notes.md [hunk 5]
```diff
@@ -14,6 +14,26 @@ title: Release notes
...
+
+### Fixed issues
+
+* The Angel Service can now correctly resolve argument paths when the absolute path of the Angel Service JAR file contains spaces.
+* Users can now be reset in the Auth Service.
+
+### Upgraded dependencies
+
+* Bouncy Castle has been upgraded from 1.75 to 1.78.1.
+* CENM now supports JDK Azul 8u422 and Oracle JDK 8u422.
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/release-notes.md:35]: just say:
> CENM now supports JDK Azul 8u422

Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/release-notes.md:35]: Jakub is now going to upload Oracle JDK 8u421 to CRAFT. Leaving as is.

Comment (tomstark99) [reply content/en/platform/corda/1.6/cenm/release-notes.md:35]: Sorry this should be Oracle 8u421 not 8u422

**Review by tomstark99 — APPROVED** [review]
LGTM (one final time :)

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**